test/numerical/TestConv.cpp

```diff
@@ -36,8 +36,8 @@ bool isOMConvTheSameAsNaiveImplFor(const int N, const int CIn, const int COut,
     const int H, const int W, const int kH, const int kW, int pHBegin,
     int pHEnd, int pWBegin, int pWEnd, const ConvAutoPad autoPad) {
   static int testNum = 0;
-  printf("attempt %d with N %d, Cin %d, Cout %d, H %d, W %d, kH %d, kW %d, "
-         "pHBegin %d, pHEnd %d, pWBegin %d, pWEnd %d, autopad %s, isDynamic "
+  printf("attempt %d with N %d, Cin %d, Cout %d, input [%d, %d], "
+         "kernel [%d, %d], pads [%d, %d, %d, %d], autopad %s, isDynamic "
```

Comment:
I like the previous one where we know which pad is for H/W and begin/end.

Reply:
Fixed. Thank you.